src/handlers/actions/moveFunds.ts

```diff
@@ -113,6 +125,9 @@ const updateExpenditureBalances = async (
   expenditure: ExpenditureFragment,
   tokenAddress: string,
   amount: string,
+  colonyAddress: string,
+  initiatorAddress: string,
+  isPartOfOneTxPayment: boolean,
```

Comment:
Could the arguments be transformed into an object? I think block-ingestor's missing the eslint rule on this